src/components/Modals/ExpertOverviewModal.tsx

```diff
@@ -86,12 +86,27 @@ export const ExpertOverviewModal: React.FC = () => {
     >
       <div className={classes.modal}>
         <Title size="h2">{MODAL_TITLE}</Title>
-        <Search
-          placeholder="Suche nach Expert*innen für.."
-          allowClear
-          onSearch={onSearch}
-          className={classes.search}
-        />
+        {experts && (
+          <Select
```

Comment:
The search doesn't work anymore as intended. In general I should be able to search for keywords even if they are not proposed. `Select` may not be the right tool for this purpose.